security/security.c

```diff
@@ -869,9 +866,19 @@ static void __init lsm_early_task(struct task_struct *task)
 
 #define call_void_hook(FUNC, ...) do {					\
 	switch (base_slot_idx.FUNC) {					\
+	default : {							\
```

Comment:
Let's have a comment saying "there were more callbacks registered for this hook than we had space in the static slot table - we'll fall back to calling the first entries in the hlist table before we move on to the static slots" or something